### ISSUE TITLE
Bug: TypeError: Todo is not a constructor

### ISSUE THREAD

**Issue by jasonrundell** [jasonrundell/partnerstack-challenge#4]
## Reproduce
1. `npm install`
2. `npm start`
3. Open index.html in Chrome browser

## Expected

No errors from server.js

## State

On page load in browser, server.js breaks with the following:

```
        return new Todo(title = t.title);
               ^

TypeError: Todo is not a constructor
    at firstTodos.map (E:\GitHub\partnerstack-challenge\server.js:13:16)
    at Array.map (<anonymous>)
    at Namespace.server.on (E:\GitHub\partnerstack-challenge\server.js:11:27)
    at Namespace.emit (events.js:182:13)
    at Namespace.emit (E:\GitHub\partnerstack-challenge\node_modules\socket.io\lib\namespace.js:209:10)
    at E:\GitHub\partnerstack-challenge\node_modules\socket.io\lib\namespace.js:177:14
    at process._tickCallback (internal/process/next_tick.js:61:11)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! challenge@1.0.1 start: `node server.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the challenge@1.0.1 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

**Comment by jasonrundell** [comment]
Typo in todo.js causing the issue. Merged into 1.1.0 feature

**Comment by jasonrundell** [comment]
- fixed typo in order for task to be added
- fixed duplication of tasks by clearing the list `<ul>` element first before adding the tasks to it
